examples/CMakeLists.txt

```
@@ -36,7 +36,7 @@
 cmake_minimum_required(VERSION 3.12)
 
 set (PROJECT_NAME ispc_examples)
-project(${PROJECT_NAME})
+project(${PROJECT_NAME} C CXX)
 
```

```suggestion

if(CMAKE_BUILD_TYPE)
    # Validate build type
    set(CONFIGURATION_TYPES "Debug;Release;RelWithDebInfo")

    string(FIND "${CONFIGURATION_TYPES}" "${CMAKE_BUILD_TYPE}" MATCHED_CONFIG)
    if (${MATCHED_CONFIG} EQUAL -1)
         message(FATAL_ERROR "CMAKE_BUILD_TYPE (${CMAKE_BUILD_TYPE}) allows only the following values: ${CONFIGURATION_TYPES}")
    endif()
else(NOT CMAKE_BUILD_TYPE)
    set(CMAKE_BUILD_TYPE "Release")
    message(STATUS "Build type not specified: Use Release by default.")
endif(CMAKE_BUILD_TYPE)

```

Another side effect of moving examples to external project is loosing default build type, while it's critical to build as `Release` by default, as its used for performance evaluation. This is copy-paste of ISPC CMakeLists.txt.